### PR TITLE
Image return

### LINKED ISSUE #312: dosomething_image return
- return themed image
- add crop aspects/crop tool?
- return image url

### LINKED ISSUE #448: Campaign: Preprocess Image vars
depends on #312

### PR DESCRIPTION
**Pull request by angaither**
Update the function to return themed images.
Add images to campagin vars array
Fixes #448
Fixes #312

### CODE REVIEW

File: modules/dosomething/dosomething_image/dosomething_image.module [hunk 1]
```diff
@@ -8,28 +8,33 @@ include_once 'dosomething_image.features.inc';
 
 /**
  * Returns an image src for any image field
- * @param array $field
- *  The image field to grab the image for.
+ * @param array $nid
```
Comment (aaronschachter) [inline modules/dosomething/dosomething_image/dosomething_image.module]: Do you mean `@param int $nid` ?  If so, line comment below too.

File: modules/dosomething/dosomething_image/dosomething_image.module [hunk 2]
```diff
@@ -8,28 +8,33 @@ include_once 'dosomething_image.features.inc';
...
+      $path =  $node->field_image_square[LANGUAGE_NONE][0]['uri'];
       break;
   }
-  return;
+
+  $img = (theme('image_style', array('style_name' => $style, 'path' => $path)));
+  return $img;
 }
 
 //@todo: build this out, return themed image, image urls as needed.
```
Comment (aaronschachter) [inline modules/dosomething/dosomething_image/dosomething_image.module:40]: Is this still relevant?

Comment (angaither) [inline modules/dosomething/dosomething_image/dosomething_image.module:40]: Yup, this function is still pretty weak... we will probably need more stuffs later.

**Comment by aaronschachter** [comment]
:+1:

**Pull request merged**